src/app/sensors/list/device-list.scss

```diff
@@ -33,6 +34,12 @@
     }
 }
 
+table {
+    button[name=remove-device] {
+        padding: 0 15px;
+    }
+}
```

Comment:
we shouldn`t use tag selectors in styles, you can find more info in [markup styleguide](https://github.com/garage-it/SmartHouse-frontend/wiki/HTML-style-guide)